apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -930,5 +930,314 @@ module.exports = {
+                              name: "width",
+                              widget: "string",
+                              required: false,
+                            },
+                          ],
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Main Navigation Links",
+                  name: "main-navigation-links",
+                  label_singular: "Main Navigation Link",
+                  widget: "list",
+                  summary: "{{link.content}}",
+                  fields: [
+                    {
+                      label: "Link",
+                      name: "link",
+                      widget: "object",
+                      fields: [
+                        {
+                          label: "Content",
+                          name: "content",
+                          widget: "string",
+                        },
+                        {
+                          label: "Href",
+                          name: "href",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Secondary Navigation Links",
+                  name: "secondary-navigation-links",
+                  label_singular: "Secondary Navigation Link",
+                  widget: "list",
+                  summary: "{{link.content}}",
+                  fields: [
+                    {
+                      label: "Link",
+                      name: "link",
+                      widget: "object",
+                      fields: [
+                        {
+                          label: "Content",
+                          name: "content",
+                          widget: "string",
+                        },
+                        {
+                          label: "Href",
+                          name: "href",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Subscription Link",
+                  name: "subscription-link",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Link",
+                      name: "link",
+                      widget: "object",
+                      fields: [
+                        {
+                          label: "Title",
+                          name: "title",
+                          widget: "string",
+                        },
+                        {
+                          label: "Href",
+                          name: "href",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
```

Comment:
```suggestion
                  widget: "code",
                  allow_language_selection: false,
                  default_language: "html"
```

We just need the code (similar to [this](https://github.com/CodeForAfrica/ui/blob/99af7c008a5834ba713d095398ec3eadc6b34210/apps/codeforafrica/src/lib/index.js#L130-L144))